target/linux/armvirt/config-5.15

```diff
@@ -1,54 +1,135 @@
+CONFIG_64BIT=y
```

Comment:
This will also activate `CONFIG_64BIT` on the armvirt/32 target. 
I think you should refresh the `target/linux/armvirt/config-5.15` when you select the 32 bit arm target.

Reply:
Oops, the kernel_menuconfig for arm64 leaked back into the top level. Fixed